package.json

```diff
@@ -64,6 +64,7 @@
     }
   },
   "volta": {
-    "node": "20.9.0"
+    "node": "20.9.0",
+    "pnpm": "8.10.2"
```

Comment:
This was bugging me.

Reply:
This doesn't play nicely with folks who don't have pnpm support explicitly turned on (`VOLTA_FEATURE_PNPM=1`) https://github.com/volta-cli/volta/issues/737#issuecomment-1442296407